### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by lizrice** [aquasecurity/bench-common#9]
Failure output from unit tests:

```
 go test ./...
# github.com/aquasecurity/bench-common/auditeval
auditeval/test_test.go:52:10: invalid operation: res != c.want (mismatched types *testOutput and bool)
FAIL    github.com/aquasecurity/bench-common/auditeval [build failed]
```

**Comment by yoavAqua** [comment]
Still relevant?

**Comment by lizrice** [comment]
If the go tests pass then this can be closed (not super easy for me to try
this right now). I don't recall whether any changes would have addressed
this.

**Comment by yoavAqua** [comment]
All tests passed, this issue was fixed in commit 2414179cd20dcd28dbe1e7402a8d9578b33cb2e0.
